_layouts/download.html

```diff
@@ -175,8 +175,8 @@ <h2>Supported platforms</h2>
 				<span>Linux</span>
 			</a>
 			<a href="/download/macos" class="tab title-font {% if page.platform == 'MacOS' %} active {% endif %}">
```

Comment:
```suggestion
			<a href="/download/macos" class="tab title-font {% if page.platform == 'macOS' %} active {% endif %}">
```
This one probably should be changed as well.

Reply:
I saw that, but it's not user-visible text. Also, I'm not sure what page.platform returns, and whether the comparison is case-sensitive. If someone knows for sure, feel free to change!

Reply:
Actually, it already returns `macOS`, so this indeed needs to be changed. On both pages as well.